src/util/CookieUtil.js

```diff
@@ -10,35 +10,26 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-define(['okta', 'jquery.cookie'], function (Okta) {
+define(['shared/util/Cookie'], function (Cookie) {
```

Comment:
Is there a reason we can't pull it in from the Okta object? If we pull from 'shared/util/Cookie', we might be duplicating it in the final bundle.

```javascript
define(['okta', function (Okta) {
  var Cookie = Okta.internal.util.Cookie;
  // etc
});
```